src/test/java/com/synopsys/integration/alert/performance/MsTeamsPerformanceTest.java

```diff
@@ -0,0 +1,73 @@
+package com.synopsys.integration.alert.performance;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.JsonObject;
+import com.synopsys.integration.alert.channel.msteams.descriptor.MsTeamsDescriptor;
+import com.synopsys.integration.alert.common.descriptor.ProviderDescriptor;
+import com.synopsys.integration.alert.common.descriptor.config.ui.ChannelDistributionUIConfig;
+import com.synopsys.integration.alert.common.descriptor.config.ui.ProviderDistributionUIConfig;
+import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
+import com.synopsys.integration.alert.common.enumeration.FrequencyType;
+import com.synopsys.integration.alert.common.enumeration.ProcessingType;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.common.rest.model.JobFieldModel;
+import com.synopsys.integration.alert.performance.model.MsTeamsPerformanceProperties;
+import com.synopsys.integration.exception.IntegrationException;
+
+public class MsTeamsPerformanceTest extends IntegrationPerformanceTest {
+    private final static String MS_TEAMS_PERFORMANCE_JOB_NAME = "MsTeams Performance Job";
+    private final MsTeamsPerformanceProperties msTeamsProperties = new MsTeamsPerformanceProperties();
+
+    @Test
+    @Ignore
+    public void testMsTeamsJob() throws Exception {
+        runTest();
+    }
+
+    @Override
+    public String createJob() throws IntegrationException {
```

Comment:
Once again, I'm wondering if it would be better to have a class `PerformanceTestJobCreator` that handled this per channel. I think we should try to move away from test inheritance.